### PR TITLE
[v6r19] EL mapping change and change the colour map of the quality plots

### PR DESCRIPTION
**Pull request by zmathe**
BEGINRELEASENOTES

*Core:
CHANGE: Change the colour map of the Quality plots
*MonitoringSystem
CHANGE: Elasticsearch 6 does not support multiple types, only one type is created instead.
ENDRELEASENOTES

### CODE REVIEW

File: Core/Utilities/Graphs/QualityMapGraph.py [hunk 1]
```diff
@@ -171,7 +171,7 @@ def draw( self ):
     setp( self.ax.get_yticklines(), markersize = 0. )
 
     cax, kw = make_axes( self.ax, orientation = 'vertical', fraction = 0.07 )
-    cb = ColorbarBase( cax, cmap = cm.RdYlGn, norm = self.norms,  #pylint: disable=no-member
+    cb = ColorbarBase( cax, cmap = self.cmap, norm = self.norms,  #pylint: disable=no-member
```
Comment (fstagni) [inline Core/Utilities/Graphs/QualityMapGraph.py]: You can remove the `#pylint: disable` now

**Comment by fstagni** [comment]
Not necessary for v6r19, but you can apply also pep8 style.

File: Core/Utilities/Graphs/QualityMapGraph.py (hunk 1 above)
Comment (zmathe) [reply Core/Utilities/Graphs/QualityMapGraph.py]: done

**Comment by zmathe** [comment]
I have to change the setup of my eclipse in order to apply pep8...

**Pull request merged**